usaspending_api/common/management/commands/matview_runner.py

```diff
@@ -81,22 +81,26 @@ def create_views(self):
         tasks = []
         for matview, config in self.matviews.items():
             logger.info("Creating Future for {}".format(matview))
-            sql = open(str(self.matview_dir / config["sql_filename"]), "r").read()
```

Comment:
If you wanted to, this (and the others) could be simplified as such:
```
sql = (self.matview_dir / config["sql_filename"]).read_text()
```

Reply:
Thanks Kirk. I think this is the 2nd time you've tried to educate me on this